### PR TITLE
docs: JUnit Processing and Visualization

### PR DESCRIPTION
**Pull request by devcatalin**
This PR adds a new page `Test Workflows Reports: JUnit Processing and Visualization` explaining how we use JUnit reports for visualization.

### CODE REVIEW

File: docs/articles/test-workflows-reports.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Test Workflows Reports: JUnit Processing and Visualization
```
Comment (olensmar) [inline docs/articles/test-workflows-reports.md:1]: Perhaps "JUnit Report Processing and Visualisation" is enough!?

File: docs/articles/test-workflows-reports.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Testkube provides comprehensive support for processing and visualizing JUnit XML reports generated during Test Workflows.  
+This page explains how Testkube handles these reports and presents the information to users.
+
+## JUnit Report Extraction
+
+Testkube automatically scans all artifacts for `.xml` files that are valid [JUnit XML reports](https://github.com/testmoapp/junitxml).  
+It then parses their contents to make them available for various reporting and visualization features.
```
Comment (olensmar) [inline docs/articles/test-workflows-reports.md:9]: add a "Read More" link to https://docs.testkube.io/articles/test-workflows-artifacts#junit-report-extraction ?

**Comment by olensmar** [comment]
Looks great - added some minor suggestions!

**Review by olensmar — APPROVED** [review]

File: docs/articles/results-overview.md [hunk 3]
```diff
@@ -5,4 +5,5 @@ Testkube provides a number of features to help you troubleshoot failed tests:
 - [Logs & Artifacts](logs-and-artifacts) provide the foundation for test execution troubleshooting
 - [Log Highlighting](../testkube-pro/articles/log-highlighting) makes it easy to quickly identify key findings in test execution logs
 - [AI Analysis](../testkube-pro/articles/AI-test-insights) uses AI to analyse log outputs and pinpount key findings
-- [Cached Test Results](../testkube-pro/articles/cached-results) help you work with execution results even if the cluster your tests were run in is not available 
+- [Cached Test Results](../testkube-pro/articles/cached-results) help you work with execution results even if the cluster your tests were run in is not available
```
Comment (jfermi) [inline docs/articles/results-overview.md:8]:
```suggestion
- [Cached Test Results](../testkube-pro/articles/cached-results) help you work with execution results even if the cluster your tests were run in is not available.
```

File: docs/articles/results-overview.md [hunk 4]
```diff
@@ -5,4 +5,5 @@ Testkube provides a number of features to help you troubleshoot failed tests:
 - [Logs & Artifacts](logs-and-artifacts) provide the foundation for test execution troubleshooting
 - [Log Highlighting](../testkube-pro/articles/log-highlighting) makes it easy to quickly identify key findings in test execution logs
```
Comment (jfermi) [inline docs/articles/results-overview.md:6]:
```suggestion
- [Log Highlighting](../testkube-pro/articles/log-highlighting) makes it easy to quickly identify key findings in test execution logs.
```

File: docs/articles/results-overview.md [hunk 5]
```diff
@@ -5,4 +5,5 @@ Testkube provides a number of features to help you troubleshoot failed tests:
 - [Logs & Artifacts](logs-and-artifacts) provide the foundation for test execution troubleshooting
```
Comment (jfermi) [inline docs/articles/results-overview.md:5]:
```suggestion
- [Logs & Artifacts](logs-and-artifacts) provide the foundation for test execution troubleshooting.
```

File: docs/articles/results-overview.md [hunk 6]
```diff
@@ -5,4 +5,5 @@ Testkube provides a number of features to help you troubleshoot failed tests:
...
 - [Log Highlighting](../testkube-pro/articles/log-highlighting) makes it easy to quickly identify key findings in test execution logs
 - [AI Analysis](../testkube-pro/articles/AI-test-insights) uses AI to analyse log outputs and pinpount key findings
```
Comment (jfermi) [inline docs/articles/results-overview.md:7]:
```suggestion
- [AI Analysis](../testkube-pro/articles/AI-test-insights) uses AI to analyze log outputs and pinpoint key findings.
```

**Review by jfermi — COMMENTED** [review]
Some minor editing suggestions.

**Pull request merged**